.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+commands:
+  before_install:
+    steps:
+      - run: |
+          dnf install make -y
+
+  install:
+    steps:
+      - run:
+          name : list_folder
+          command : ls
+      - run:
+          name : install_cmd
+          command: make install
+  
+  before_script:
+    steps:
+      - run: |
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" tls $NUO_SET_TLS
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" start
+
+
+  script:
+    steps:
+      - run:
+          name: script_cmd
+          command: make test NUODB_HOME="/opt/nuodb"
+  
+  after_failure:
+    parameters:
+      when:
+        type: string
+    steps:
+      - run:
+          command: |
+            cat /var/log/nuodb/nuoadmin.log*
+            grep ssl /opt/nuodb/etc/nuoadmin.conf"
+          when: <<parameters.when>>
+
+  build:
+    steps:
+      - checkout
+      - before_install
+      - install
+      - before_script
+      - script
+      - after_failure:
+          when : "on_fail"
```

Comment:
Do you know if it a test report is created? We should publish the test report to CircleCI so that we can see which test failed. I think currently it just shows the entire run as being failed without any detail about which tests are being run.

See the following in the nuodb-control-plane CircleCI config:https://github.com/nuodb/nuodb-control-plane/blob/main/.circleci/config.yml#L175

Reply:
[Here](https://circleci.com/docs/collect-test-data/#pytest) is also an example on how to collect the reports from _pytest_.